### PR TITLE
Pubstwo-1586 - Replace current header/footers with USGS header/footer using USWDS

### PR DESCRIPTION
**Pull request by mbucknell**

### CODE REVIEW

File: server/pubs_ui/templates/base.html [hunk 1]
```diff
@@ -7,62 +7,37 @@
...
-        <script src="{{ 'vendor/system.js' | asset_url }}"></script>
+
         {% block page_style %}
             <link rel="stylesheet" href="{{ 'usgs_style.css' | asset_url }}" type="text/css" />
         {% endblock %}
-        {% block head_script %}
-        {% endblock %}
+
+        <!--[if gt IE 8]><!-->
+        <script src="{{ 'vendor/polyfill.js' | asset_url }}"></script>
```
Comment (danielnaab) [inline server/pubs_ui/templates/base.html]: I'm guessing this is needed by system.js?

**Comment by danielnaab** [comment]
I don't see the USWDS Javascript imported anywhere - maybe needs to be done in each separate bundle?

File: server/pubs_ui/templates/base.html (hunk 1 above)
Comment (mbucknell) [reply server/pubs_ui/templates/base.html]: Yes for IE. This is the fix that Aaron implemented. I added it to the base template that is inherited by all apps so now all pages should "work" in IE.

**Comment by mbucknell** [comment]
It isn't. I'm not sure if we need it. I was never sure why we brought it in. Do we need it?

**Comment by mbucknell** [comment]
I should just bring in the USWDS javascript so that it is there when we do want to use it. I will do that before merging.

**Comment by danielnaab** [comment]
I think just the accordion, which is used in the header. Technically, could just import that one module..

**Comment by mbucknell** [comment]
I imported the whole thing. Created a common.js module which imports uswds. It is loaded in the pubs_ui/templates/base.html so that all pages get the module.

**Pull request merged**